### PR TITLE
fix wrong password bug, fixes #400

### LINKED ISSUE #400: If I enter a wrong password, then I can't try again without exiting module and coming back to it.
image

### PR DESCRIPTION
**Pull request by NemTechCN**
change #400 issue

### CODE REVIEW

File: src/views/service/mosaic/mosaic-function/mosaic-transaction/MosaicTransactionTs.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ import {formDataConfig, Message, DEFAULT_FEES, FEE_GROUPS} from '@/config'
 import {StoreAccount, AppWallet, DefaultFee} from "@/core/model"
 import {NETWORK_PARAMS} from '@/core/validation'
 import {createBondedMultisigTransaction, createCompleteMultisigTransaction} from '@/core/services'
+import { LoadingBar } from 'iview'
```
Comment (decentraliser) [inline src/views/service/mosaic/mosaic-function/mosaic-transaction/MosaicTransactionTs.ts]: this line is not used

Comment (decentraliser) [reply src/views/service/mosaic/mosaic-function/mosaic-transaction/MosaicTransactionTs.ts]: removed

**Comment by NemTechCN** [comment]
> This is clearing a bug but this does not fix the issue opened by Jeff

the error is that a user ano not reopen modal after input error password , what do you think of this issue

**Comment by decentraliser** [comment]
for some reason it now works on subnamespace creation

let's let it like that and see if there are some problems afterwards

**Pull request merged**